src/main/java/edu/kis/powp/jobs2d/command/history/HistoryCommandObject.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.powp.jobs2d.command.history;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+
+public class HistoryCommandObject {
+    private String name;
+    private String date;
+    private DriverCommand command;
+    private Job2dDriver driver;
```

Comment:
I would prefer commands in the history not to be coupled with driver, so they can be reused in other context.